src/mock-context.js

```diff
@@ -18,6 +18,7 @@ export function mockContext(url: string, options: *): Context {
     const {path} = parsedUrl;
     parsedUrl.path = parsedUrl.pathname;
     parsedUrl.url = path;
+    //$FlowFixMe
     return {parsedUrl};
```

Comment:
Some elements are missing from the _parsedUrl_ if we expect this to return a _Context_ object.  For now, I think it is sufficient to ignore this warning until we are in a state to update all of `fusion-test-utils`.